auto/run.sh

```diff
@@ -37,68 +37,70 @@ pushd Pillow
     "$@"
   }
 
+  testsuite_run=../../testsuite/run.py
```

Comment:
Actually, I think something simpler and shorter is preferable. Maybe just `$RUN`?

Reply:
Okay, sure. Updated.